### PR TITLE
Fix typographical and grammatical errors and inconsistencies

### PR DESCRIPTION
**Pull request by whitequark**
The initial list of hits was generated with the codespell command
below, and each hit was evaluated and fixed manually while taking
context into consideration.

    DIRS="kernel/ frontends/ backends/ passes/ techlibs/""
    codespell $DIRS -S *.o -L upto,iff,thru,synopsys,uint

More hits were found by looking through comments and strings manually.

### CODE REVIEW

**Comment by whitequark** [comment]
Also, there are a number of typos in ezsat and subcircuit libraries, but I wasn't able to find the upstream source for these. Do you have a repository somewhere I should add a PR against, or should I just fix them here?

File: backends/table/table.cc [hunk 1]
```diff
@@ -109,7 +109,7 @@ struct TableBackend : public Backend {
 				else if (cell->output(conn.first))
 					*f << "out" << "\t";
 				else
-					*f << "unkown" << "\t";
+					*f << "unknown" << "\t";
```
Comment (whitequark) [inline backends/table/table.cc]: This seems like a bug.

File: frontends/verific/verificsva.cc [hunk 2]
```diff
@@ -827,9 +827,9 @@ struct SvaFsm
 
 			for (auto &it : nodes[i].edges) {
 				if (it.second != State::S1)
-					log("          egde %s -> %d\n", log_signal(it.second), it.first);
+					log("          edge %s -> %d\n", log_signal(it.second), it.first);
```
Comment (whitequark) [inline frontends/verific/verificsva.cc]: These four seem like a bug.

File: passes/cmds/setundef.cc [hunk 3]
```diff
@@ -137,7 +137,7 @@ struct SetundefPass : public Pass {
 		log("        replace with $anyconst drivers (for formal)\n");
 		log("\n");
 		log("    -random <seed>\n");
-		log("        replace with random bits using the specified integer als seed\n");
+		log("        replace with random bits using the specified integer seed\n");
```
Comment (whitequark) [inline passes/cmds/setundef.cc]: Not sure what "als" means here, I assumed a copy-paste error.

Comment (cliffordwolf) [reply passes/cmds/setundef.cc]: Oops, yes. Should have been "as".  :)

**Comment by cliffordwolf** [comment]
> Also, there are a number of typos in ezsat and subcircuit libraries, but I wasn't able to find the upstream source for these. Do you have a repository somewhere I should add a PR against, or should I just fix them here?

There is no other repo for those two libraries. This is already upstream. They are just in `libs/` because they are written in a manner that resembles a self-contained library.

**Comment by whitequark** [comment]
All fixed.

File: passes/cmds/select.cc [hunk 4]
```diff
@@ -987,7 +987,7 @@ struct SelectPass : public Pass {
 		log("list of selected objects.\n");
 		log("\n");
 		log("Note that many commands support an optional [selection] argument that can be\n");
-		log("used to YS_OVERRIDE the global selection for the command. The syntax of this\n");
+		log("used to override the global selection for the command. The syntax of this\n");
```
Comment (cliffordwolf) [inline passes/cmds/select.cc]: Hahaha!  Looks like this was collateral damage when we switched all users of the C++ keyword `overwrite` to the `YS_OVERWRITE` macro.

Comment (whitequark) [reply passes/cmds/select.cc]: Yes. I was greatly amused by this one.

Comment (daveshah1) [reply passes/cmds/select.cc]: https://www.newscientist.com/letter/mg16221857-700-ocralomania/

**Pull request merged**